.github/workflows/macos.yml

```diff
@@ -18,10 +18,10 @@ jobs:
         conf:
           - name: Clang
             flags: -c clang
-            max_warnings: 1
+            max_warnings: 0
           - name: GCC-9
```

Comment:
Another one bites the dust 💨  
On track to zero out gcc 9 by the time 0.76 is ready.